### PR TITLE
BZ1247145: Fly-over notifications not shown for copy/rename/delete done via Project Explorer

### PR DESCRIPTION
**Pull request by manstis**
See https://bugzilla.redhat.com/show_bug.cgi?id=1247145

### CODE REVIEW

**Comment by manstis** [comment]
@ederign I've written (with your help) tests to verify the NotificationEvent is raised when items are copied, renamed or deleted... thought you'd be best to review and merge :)

File: kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/pom.xml [hunk 1]
```diff
@@ -132,6 +132,19 @@
...
+    <dependency>
+      <groupId>com.google.gwt.gwtmockito</groupId>
+      <artifactId>gwtmockito</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.uberfire</groupId>
+      <artifactId>uberfire-testing-utils</artifactId>
+      <scope>testing</scope>
```
Comment (ederign) [inline kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/pom.xml]: <scope>test</scope>?

Comment (manstis) [inline kie-wb-common-screens/kie-wb-common-project-explorer/kie-wb-common-project-explorer-client/pom.xml]: Hehe... why limit the scope to test when testing is so much more fun :-)

**Comment by ederign** [comment]
Please fix the scope (if necessary) and we're good to go!

**Comment by manstis** [comment]
Fixed.

**Comment by manstis** [comment]
@ederign You really should merge locally using the CLI and rebase the "feature branch" on master to keep the commit history linear.. look at the mess you've made (gitg etc) ;)

**Comment by ederign** [comment]
Oh... it was my first one. Should I revert it?

**Comment by manstis** [comment]
@ederign nah.. you're not the first (whoever did the "JBPM-4625 Generation of JPA enabled models" merge before yours did the same.. @wmedvede was this you working weekends?)

**Comment by ederign** [comment]
OK sorry about it! I'll follow your instructions in the next one!

**Comment by manstis** [comment]
@ederign the cherry-pick to 6.3.x is also a bit confusing when the merge history isn't linear :(

**Comment by ederign** [comment]
:(

**Pull request merged**